docs/USAGE.md

```diff
@@ -29,26 +33,61 @@ followed a manual installation as per [here](INSTALL.md)).
 
    j. [What is the Gap Limit](#gaplimit)
 
-3. [Try out a coinjoin; using sendpayment.py](#try-coinjoin)
+4. [Try out a coinjoin; using sendpayment.py](#try-coinjoin)
+
+5. [Running a "Maker" or "yield generator"](#run-maker)
+
+6. [Running the tumbler script to boost privacy of owned coins](#run-tumbler)
+
+<a name="data" />
+
+## Managing your Joinmarket data
 
-4. [Running a "Maker" or "yield generator"](#run-maker)
+First thing to do: go into `scripts/`, and run:
+
+        (jmvenv)$ python wallet-tool.py generate
 
-5. [Running the tumbler script to boost privacy of owned coins](#run-tumbler)
+This *should* quit with an error, because the connection to Bitcoin Core is not configured; we'll cover that in the next section.
+However, this first run will have automatically created a data directory.
+Locate the newly created file `joinmarket.cfg` which will be in your user home directory under `.joinmarket/`.
+So on Linux you should find it under `/home/username/.joinmarket/joinmarket.cfg`, and similarly for MacOS and Windows.
+You should see the following files and folders for an initial setup:
+
+```
+.joinmarket/
+    joinmarket.cfg
+    logs/
+    wallets/
+    cmtdata/
+```
+
+`joinmarket.cfg` is the main configuration file for Joinmarket and has a lot of settings, several of which you'll want to edit or at least examine.
+This will be discussed in several of the sections below.
+The `wallets/` directory is where wallet files, extension (by default) of `.jmdat` are stored after you create them. They are encrypted and store important information; without them, it is possible to recover your coins with the seedphrase, but can be a hassle, so keep the file safe.
+The `logs/` directory contains a log file for each bot you run (Maker or Taker), with debug information. You'll rarely need to read these files unless you encounter a problem; deleting them regularly is recommended (and never dangerous). However there are other log files kept here, in particular one called `yigen-statement.csv` which records all transactions your Maker bot does over time. This can be useful for keeping track. Additionall, tumbles have a `TUMBLE.schedule` and `TUMBLE.log` file here which can be very useful; don't delete these.
+The `cmtdata/` directory stores technical information that you will not need to read.
+
+<a name="portability" />
+
+### Portability
+
+It is possible to use a different data directory than the default mentioned above (`~/.joinmarket/` or equivalent). To do this, run any Joinmarket script (see below for descriptions of the main ones) with the flag `--datadir=/my/data/directory`.
+Then the above directory structure will be created there. If you move a wallet file from one directory to another it will work fine, as long as you are using the same instance of Bitcoin Core.
```

Comment:
"as long as you are using the same instance of Bitcoin Core and don't change `rpc_wallet_file` setting"?

Reply:
Not sure it's worth the extra words tbh.